crypto/bn/bn_rsa_fips186_4.c

```diff
@@ -357,7 +357,7 @@ int ossl_bn_rsa_fips186_4_derive_prime(BIGNUM *Y, BIGNUM *X, const BIGNUM *Xin,
              *    sqrt(2) * 2^(nlen/2-1) <= Random X <= (2^(nlen/2)) - 1.
              */
             if (!BN_priv_rand_range_ex(X, range, 0, ctx) || !BN_add(X, X, base))
-                goto end;
+                goto err;
```

Comment:
Good catch,,

Not sure if it is possible in practice.. BN_priv_rand_range_ex() does 100 iterations so it would have a very very small probability that this could ever fail.